### PR TITLE
[main][fix] Contract check in SafeToken lib

### PR DESCRIPTION
**Pull request by alpanaca**
## Description
Add check if the callee is a contract in SafeToken Lib

## Why?
Security purpose

### CODE REVIEW

**Comment by almost-llamas** [comment]
:eyes:

**Comment by spicysquid168** [comment]
Nicee

**Comment by spicysquid168** [comment]
I like this

**Comment by almost-llamas** [comment]
Oh, it sync deleted msg too

File: contracts/8.10/SafeToken.sol [hunk 1]
```diff
@@ -29,6 +29,7 @@ library SafeToken {
   ) internal {
     // bytes4(keccak256(bytes('transfer(address,uint256)')));
     // solhint-disable-next-line avoid-low-level-calls
+    require(token.code.length > 0, "!contract");
```
Comment (almost-llamas) [inline contracts/8.10/SafeToken.sol:32]: Test comment

Comment (alpanaca) [reply contracts/8.10/SafeToken.sol:32]: Test Reply _(with Axolo)_

**Review by almost-llamas — APPROVED** [review]

**Comment by spicysquid168** [comment]
Looks nice

**Review by sundayalpaca — APPROVED** [review]

**Pull request merged**